trackpy/feature.py

```diff
@@ -274,24 +281,29 @@ def _refine(raw_image, image, radius, coords, max_iterations,
         mass[feat] = neighborhood.sum()
         if not characterize:
             continue  # short-circuit loop
-        Rg[feat] = np.sqrt(np.sum(r_squared_mask(radius, ndim) *
-                                  neighborhood) / mass[feat])
+        if radius[1:] == radius[:-1]:
```

Comment:
Not necessarily, as `radius` is a tuple:

    if tuple(np.arange(5)) == tuple(np.arange(5)):
        print('yo')
    yo